recipe_diagnostics/diagnostic_scripts/enso_diag1metrics.py

```diff
@@ -67,31 +67,15 @@ def plot_level1(input_data, metricval, y_label, title, dtls): #input data is 2 -
 
     return figure
 
-def lin_regress(cube_ssta, cube_nino34): #1d 
+def lin_regress(cube_ssta, cube_nino34): #1d pattern
```

Comment:
I had a look at the regression functions yesterday. The call to linregress is very inefficient.
I have started investigating the option to solve it using matrix operation. Have a look at my notebook in #28
I think it is OK for now but definitely needs to be optimised.
Note that I have some issues in that notebook, so don't assume it is working :-/